node/node_handler.go

```diff
@@ -631,3 +678,33 @@ func lookPath() (argv0 string, err error) {
 	}
 	return
 }
+
+func (node *Node) storeEpochShardState(epochShardState *types.EpochShardState) {
+	byteBuffer := bytes.NewBuffer([]byte{})
+	encoder := gob.NewEncoder(byteBuffer)
+	err := encoder.Encode(epochShardState)
+	if err != nil {
+		utils.GetLogInstance().Error("[Resharded] Failed to encode epoch shard state", "error", err)
+	}
+	err = ioutil.WriteFile("./tmp_log/epoch_shard_state"+node.SelfPeer.IP+node.SelfPeer.Port, byteBuffer.Bytes(), 0644)
```

Comment:
is it okay if the tmp_log directory doesn't exist?

Reply:
No it doesn't. Changed to store in ./epoch_shard_state